src/NServiceBus.Core/Routing/AssemblyRouteSource.cs

```diff
@@ -20,7 +20,7 @@ public AssemblyRouteSource(Assembly messageAssembly, UnicastRoute route)
         public IEnumerable<RouteTableEntry> GenerateRoutes(Conventions conventions)
         {
             var routes = messageAssembly.GetTypes()
-                .Where(t => conventions.IsMessageType(t))
+                .Where(conventions.IsMessageType)
```

Comment:
@timbussmann i guess my problem here is it makes it difficult to contribute to a repo where a PR can be blocked when it makes a change that is consistent with the rest of the codebase. if you want to take a stance on "no method groups" then u need to apply it consistently.